src/util/util.js

```diff
@@ -4,6 +4,7 @@
 const UnitBezier = require('@mapbox/unitbezier');
 const Coordinate = require('../geo/coordinate');
 const Point = require('point-geometry');
+const LngLat = require('../geo/lng_lat');
```

Comment:
I've removed the circular dependency by moving the `bestWorld` function out of util/util.